src/frontend/src/views/Upload/components/Metadata/index.tsx

```diff
@@ -154,11 +154,11 @@ export default function Metadata({
             </ContinueButton>
           </NextStepWrapper>
           <NextLink href={ROUTES.UPLOAD_STEP1} passHref>
-            <a href="passHref">
+            <Link href="passHref">
               <Button sdsType="secondary" sdsStyle="rounded">
                 Back
               </Button>
-            </a>
+            </Link>
```

Comment:
Our use of `a` tags with `button` tags inside seems strange.  Maybe we should change these to be links styled like buttons.  ...there was pushback from SDS last year about this - the argument against was that we should make links look like links, but the reality is that this is often not the case.